metrics/line-numbers/fineLineNumber.go

```diff
@@ -0,0 +1,48 @@
+package linenumbers
```

Comment:
I think this file's name has a typo.

... regardless of that, I'd like to use a different name for this package. yaml.Nodes give us access to comments and column positions along with line numbers, and this recursive is useful for getting all of that.

Protocol Buffers call this kind of information "source info" and I think that's a good pattern to follow. Let's name this package "sourceinfo" (and put it in a directory named "metrics/sourceinfo"). This file could be just "sourceinfo.go".